### PR TITLE
Vocab search language menu

### PR DESCRIPTION
**Pull request by joelit**
## Reasons for creating this PR
The search bar language menu contents were a mockup while waiting for the translation component. This PR separates the mockup translations and adds the functionality of using the vocabulary config languages for forming the search bar language menu contents.

## Description of the changes in this PR
- Added the vocabulary config languages to the SKOSMOS object for ease of testing
- Added the functionality for using the vocab.config languages when forming the search bar language menu
- Added tests for the language menu contents
## Checklist

- [x] phpUnit tests pass locally with my changes
- [x] I have added tests that show that the new code works, or tests are not relevant for this PR (e.g. only HTML/CSS changes)
- [x] The PR doesn't reduce accessibility of the front-end code (e.g. tab focus, scaling to different resolutions, use of `.sr-only` class, color contrast)
- [x] The PR doesn't introduce unintended code changes (e.g. empty lines or useless reindentation)

### CODE REVIEW

File: tests/cypress/template/vocab-search-bar.cy.js [hunk 1]
```diff
@@ -53,6 +53,23 @@ describe('Vocab search bar', () => {
...
+    it('available search languages are the ones described in the vocabulary config', () => {
+      cy.visit('/yso/en/') // go to the YSO home page in English language
+
+      // check that the vocabulary languages can be found in the search bar language dropdown menu
+      cy.window().then((win) => {
+        cy.get('#language-list .dropdown-item').then($elements => {
+          const actualLanguages = $elements.map((index, el) => Cypress.$(el).attr('value')).get();
+
+          // The expected languages should be those found in vocabulary configuration with an additional 'all' language
+          const expectedLanguages = [...win.SKOSMOS.languages, 'all'];
```
[the remark this answers is not in the document]
Comment (osma) [reply tests/cypress/template/vocab-search-bar.cy.js:65]: Relying on `win.SKOSMOS.languages` here could be problematic. What if it is not set properly based on the vocabulary configuration? Then the test could pass while the functionality is actually broken.

I suggest hardcoding the values here instead to match what is actually in the Skosmos configuration file used for the Cypress tests.

**Review by osma — CHANGES_REQUESTED** [review]
Looks good, but I suggested a couple of changes.

File: tests/cypress/template/vocab-search-bar.cy.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (joelit) [reply tests/cypress/template/vocab-search-bar.cy.js:65]: Good point!

**Review by osma — APPROVED** [review]
LGTM

**Pull request merged**